tests/data/firefox/tinderbox-builds/mozilla-central-macosx64/1425294124/firefox-25.0a1.en-US.mac.txt

```diff
@@ -0,0 +1,2 @@
+20150302030204
+https://hg.mozilla.org/mozilla-central/rev/eea6188b9b05
```

Comment:
Please put this file into https://github.com/mozilla/mozdownload/tree/master/tests/data/firefox/tinderbox-builds/mozilla-central-macosx64/1374568307